### ISSUE TITLE
Windows kubelet won't start "Failed to create listener for podResources endpoint"

### ISSUE THREAD

**Issue by PatrickLang** [kubernetes/kubernetes#78628]
**What happened**:

I replaced the kubelet on one of my 1.14 nodes with a build from master, and now I'm getting this error starting up.

Test passes since 5/31/2019 commit b7fa33ec1 have been failing on testgrid. If you look at the logs, the Windows nodes are not coming up. https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kubernetes-e2e-aks-engine-azure-master-windows/1134583855047512068

```
F0602 02:35:46.663366   12224 server.go:179] Failed to create listener for podResources endpoint: parse npipe://%5Cvar%5Clib%5Ckubelet%5Cpod-resources//./pipe/kubelet: invalid URL escape "%5C"
goroutine 156 [running]:
k8s.io/kubernetes/vendor/k8s.io/klog.stacks(0x72b9d01, 0x3, 0xc00041c280, 0xc1)
        /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/vendor/k8s.io/klog/klog.go:900 +0xb8
k8s.io/kubernetes/vendor/k8s.io/klog.(*loggingT).output(0x72b9d80, 0xc000000003, 0xc0003b6b60, 0x71c6d2d, 0x9, 0xb3, 0x0)
        /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/vendor/k8s.io/klog/klog.go:815 +0xed
k8s.io/kubernetes/vendor/k8s.io/klog.(*loggingT).printf(0x72b9d80, 0x3, 0x40f4965, 0x37, 0xc000a4bea0, 0x1, 0x1)
        /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/vendor/k8s.io/klog/klog.go:727 +0x155
k8s.io/kubernetes/vendor/k8s.io/klog.Fatalf(...)
        /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/vendor/k8s.io/klog/klog.go:1309
k8s.io/kubernetes/pkg/kubelet/server.ListenAndServePodResources(0xc0002a9740, 0x3e, 0x9a1abb0, 0xc000289300, 0x9a1abd0, 0xc00078d320)
        /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/kubelet/server/server.go:179 +0x175
k8s.io/kubernetes/pkg/kubelet.(*Kubelet).ListenAndServePodResources(0xc0002c0900)
        /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/kubelet/kubelet.go:2216 +0x220
created by k8s.io/kubernetes/cmd/kubelet/app.startKubelet
        /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/cmd/kubelet/app/server.go:1103 +0x123

```

**What you expected to happen**:
No errors starting up kubelet

**How to reproduce it (as minimally and precisely as possible)**:
Build master, then try to run the kubelet on Windows

**Anything else we need to know?**:

/sig windows

**Environment**:
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration: Azure
- OS (e.g: `cat /etc/os-release`): Windows Server 2019
- Kernel (e.g. `uname -a`): 10.0.17763.529
- Install tools: aks-engine

**Comment by PatrickLang** [comment]
cc @adelina-t 
/priority critical-urgent

**Comment by PatrickLang** [comment]
/release v1.15

Release team - it looks like a recent checkin broke windows nodes. We will need a fix here for 1.15

**Comment by adelina-t** [comment]
Ok, so the problem here is a little more complicated and we're actually seeing a bug that is actually in the code for quite some time, just that we never used that particular code branch.

But first:
**TL;DR: Quick fix is to start the kubelet on Windows with: --feature-gates="KubeletPodResources=false"**

Now, as to why this is happening:

**Comment by yujuhong** [comment]
GCE tests failing too https://testgrid.k8s.io/google-windows#windows-gce
/cc @pjh @mtaufen

**Comment by PatrickLang** [comment]
I'm trying a quick `s/\\/\//` to see if anything else is broken. There are other functions in that same file doing that substitution already so it may make sense to use that as a workaround until a better solution is available.

**Comment by PatrickLang** [comment]
Still working on it - looks like the path to create the directories is also busted.

With 91a29c0 still getting `F0603 19:57:15.224243  285656 server.go:181] Failed to create listener for podResources endpoint: open //./var/lib/kubelet/pod-resources//./pipe/kubelet: The system cannot find the path specified.`
